CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
   between `chrono::DateTime<Utc>` and `chrono::DateTime<FixedOffset>` and Ruby
   `Time` objects.
 - `typed_data::Writebarrier::writebarrier` and `writebarrier_unprotect`.
+- Bug fixes for `IntoValue` for `chrono::DateTime<Utc>` and
+  `chrono::DateTime<FixedOffset>`
```

Comment:
You can skip this, the changelog only needs to describe differences from the last release.

Reply:
Sounds good.

I noticed some of my checkpoint commits made it into `main` ("Reviewer comments", etc.).  Should I be amending and force pushing to my topic branch to keep a single atomic commit before the PR is merged? Or perhaps you prefer non-atomic commits in main.  Let me know if I should do anything specific here.

Reply:
I'm not that fussed about how other people structure their commits. I'll merge the PR with the default "Create a merge commit" option on GitHub. If you want a chance to get commits how you like before I merge, just let me know and I can hold off on merging until you give me the ok.

Reply:
Should be good to go now, apart from any additional changes you'd like.